src/general_listener.c

```diff
@@ -667,6 +667,69 @@ void peak_general_listener_attach()
     }
 }
 
+static FILE* peak_stats_csv_open(void) {
+    const char* path = "peak_stats.csv";
```

Comment:
We probably should make the name user definable. Also, we should probably add a PID to the default name here such that multiple runs will not collide.